tests/unit/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import ops.testing
+
+ops.testing.SIMULATE_CAN_CONNECT = True
```

Comment:
I think you can move this to `conftest.py`

Reply:
done